custom_components/kumo/climate.py

```diff
@@ -223,7 +253,7 @@ def _update_hvac_action(self):
 
     @property
     def hvac_modes(self):
-        """Returns the list of available operation modes."""
+        """Return list of available operation modes."""
```

Comment:
Why are some of these properties "Return the..." and some "Return..." ?

Reply:
The error I received was "first line should be in imperative mood"

Reply:
Right, I'm asking about missing "the"

Reply:
My bad... Was basically flying blind until the errors stopped showing up. May have deleted more than I should have ..

Reply:
fixed in https://github.com/dlarrick/hass-kumo/pull/19/commits/2fb891201a5a811c70d6ff9a90643e4fa678393d